### DISCUSSION TITLE
Allow hiding the search bar in the checkbox-tree interface

### DISCUSSION THREAD

**Discussion by u12206050** [directus/directus#14673] [Feature Requests]
No  matter how many or how few choices one has in the checkbox tree, the search box is always being shown.

![Screenshot 2022-07-26 at 11 05 45](https://user-images.githubusercontent.com/6641242/181014526-de94d634-85ff-4ca5-aaa6-675072d91956.png)

### To Reproduce

Create checkbox tree, and view the form in item create or update.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
